### DISCUSSION TITLE
how to change the track

### DISCUSSION THREAD

**Discussion by Popunia** [SammyIAm/Moppy2#218] [Q&A] [answered]
maybe this is a stupid question. after searching for the answer in the wiki, I found nothing.(or didn't understand anything) How can I make it so that, for example, a disk drive connected to pins 2 and 3 plays a midi track, for example 3? The only option I figured out was to rearrange the pins. But it's long and inconvenient.
thanks

**Comment by samphonic** [comment, marked as answer]
You can change which drive plays which tracks using the Mappers at the bottom of the GUI. The drive attached to pins 2 and 3 is Sub-Address 1 (4/5 is sub-address 2, etc.). So if you wanted track 3 to play on sub-address 1, you would change "Condition" to `c==3` (only track 3) and change "Sub Address" to `1` (plays on the first drive).

**Reply by Popunia** [reply]
Thank you!
